src/main/java/org/folio/consortia/service/FolioTenantService.java

```diff
@@ -39,6 +53,11 @@ protected void afterTenantUpdate(TenantAttributes tenantAttributes) {
     try {
       contextHelper.registerTenant();
       kafkaService.createKafkaTopics();
+      var context = contextHelper.getSystemUserFolioExecutionContext(folioExecutionContext.getTenantId());
+      if (Objects.nonNull(context)) {
+        runInFolioContext(context, () ->
+          customFieldService.createCustomField(createCustomFieldObject()));
```

Comment:
What if custom field already exists and we just re-enabling the tenant?
We should check if this field exsits by name

Reply:
fixed